src/components/Map/Map.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import GoogleMapReact from 'google-map-react';
+import MenuIcon from '@material-ui/icons/Menu';
+import PropTypes from 'prop-types';
+import PlaceIcon from '@material-ui/icons/Place';
+
+class Map extends Component {
+	render() {
+		return (
+			<div style={{ height: 'calc(100vh - 47.4px)', width: '100%' }}>
+				<GoogleMapReact
+					bootstrapURLKeys={{ key: process.env.REACT_APP_GOOGLE_API_KEY }}
+					defaultCenter={{ lat: 48.462, lng: -123.313 }}
+					defaultZoom={14}
+					options={function(maps) {
+						return { mapTypeId: 'satellite' };
+					}}
+				>
+					{this.props.deerStates.map((element, index) => {
+						if (element['state'] || this.props.showAll) {
+							return element['last_location'].map(pos => {
+								console.log(pos);
+								return (
+									<PlaceIcon
+										style={{ color: element['colour'] }}
```

Comment:
Since you're switching from MaterialUI to antd, you'll need to eliminate `PlaceIcon` and the two imports at L3 and L5. You can use the `environment` icon from antd to do the same job. I replaced it like this:

```js
<Icon type="environment"
    theme="filled"
    style={{ color: element['colour'] }}
    lat={pos[0]}
    lng={pos[1]}
    text="My Marker"
/>
```

It produces really small icons, so you'll need to modify the size to match accordingly. Here's what the above code looks like:
<img width="1434" alt="Screen Shot 2019-07-13 at 2 29 08 PM" src="https://user-images.githubusercontent.com/11711350/61176725-aa1aff80-a57a-11e9-9b65-6fec88634d85.png">

Reply:
Oh, I forgot! When removing the imports for MaterialUI icons, you'll want to import icons from antd:

`import { Icon } from 'antd';`

Reply:
Oh sick! Thank you!!